compressor/management/commands/compress.py

```diff
@@ -211,44 +211,54 @@ def compress(self, log=None, **options):
                       "\n\t".join((t.template_name
                                    for t in compressor_nodes.keys())) + "\n")
 
+        contexts = settings.COMPRESS_OFFLINE_CONTEXT
+        if isinstance(contexts, six.string_types):
+            module, function = settings.COMPRESS_OFFLINE_CONTEXT.rsplit('.', 1)
```

Comment:
I'm not sure if it's much better than your version, but we have a `get_mod_func` in `compressor.utils` that does this.